sql/feedback.sql

```diff
@@ -0,0 +1,31 @@
+DROP TABLE IF EXISTS complaint_emails;
+DROP TABLE IF EXISTS complaint_messages;
+DROP TABLE IF EXISTS complaint_info;
+
+CREATE TABLE complaint_info (
+  complaint_id INT(11) NOT NULL AUTO_INCREMENT,
+  subject VARCHAR(50) DEFAULT NULL,
+  status VARCHAR(36) DEFAULT NULL,
+  uuid BINARY(16) NOT NULL,
+  org VARCHAR(10) NOT NULL,
+  PRIMARY KEY (complaint_id),
+  UNIQUE KEY (uuid)
+);
+
+CREATE TABLE complaint_messages (
+  complaint_id INT(11) DEFAULT NULL,
+  time TIMESTAMP,
+  message TEXT,
+  poster VARCHAR(50) DEFAULT NULL,
+  message_id INT(11) NOT NULL AUTO_INCREMENT,
+  org VARCHAR(10) NOT NULL,
```

Comment:
Are all `complaint_messages` attached to a `complaint_info`? If so, seems like `org` is redundant.